src/property-dictionary.js

```diff
@@ -78,8 +78,7 @@
     return (lengthValue.type == 'percent');
   };
 
-  PropertyDictionary.prototype.
-      isValidKeyword = function(property, styleValueString) {
+  PropertyDictionary.prototype.isValidKeyword = function(property, styleValueString) {
```

Comment:
Removed newline: I find this style more readable.

Also in this function, we should always return true for globally valid keywords (e.g. initial, inherit, etc)